fe/fe-core/src/test/java/org/apache/doris/analysis/PartitionPruneTestBase.java

```diff
@@ -29,8 +29,6 @@ public abstract class PartitionPruneTestBase extends TestWithFeService {
 
     protected void doTest() throws Exception {
         for (RangePartitionPruneTest.TestCase testCase : cases) {
-            connectContext.getSessionVariable().partitionPruneAlgorithmVersion = 1;
```

Comment:
partition prune V1 is to be deleted, and there are some bug. So remove it from test cases